app/assets/stylesheets/govuk_publishing_components/components/_step-by-step-nav.scss

```diff
@@ -63,15 +63,116 @@ $top-border: solid 1px govuk-colour("mid-grey", $legacy: "grey-3");
   padding: 3px 3px 0 0;
 }
 
+.gem-c-step-nav____title-text-focus {
+  margin-bottom: 13px;
+  display: inline-block;
+}
+
+// Create Chevron icon aligned with text
+.gem-c-step-nav__chevron {
+  box-sizing: border-box;
+  display: inline-block;
+  position: relative;
+  // Set size using rems to make the icon scale with text if user resizes text in their browser
+  width: govuk-px-to-rem(20px);
+  height: govuk-px-to-rem(20px);
+  border: govuk-px-to-rem(1px) solid;
+  border-radius: 50%;
+
+  vertical-align: text-top;
+
+  // Reduce size, alter alignment
+  .gem-c-step-nav--large & {
+    vertical-align: top;
+  }
+
+  // IE8 fallback of icon
+  @include govuk-if-ie8 {
```

Comment:
I was checking this in browserstack and it looks like these changes have completely broken this component in IE8 - it doesn't appear.

![Screenshot 2022-02-02 at 09 35 49](https://user-images.githubusercontent.com/861310/152128738-6520df77-bdaa-476b-84da-239f0b35b19e.png)

Also in IE9, the styling is a bit broken (current version looks very close to modern Chrome).

![Screenshot 2022-02-02 at 09 36 25](https://user-images.githubusercontent.com/861310/152128813-5a6bd115-50ed-413f-a4f6-3a4c7016febd.png)


Side point, is putting an IE8 fallback in worth the effort? With the SVG on IE8 the icon simply didn't appear, but the step nav was still usable.

Reply:
>is putting an IE8 fallback in worth the effort? 

In this context you're right, I should remove it as we don't support 8 so why is it there? This was ported from the DS work as it had to support IE8.

🤔 IE8/9 have existing issues (below) - however I can resolve to resolve functional support.

<img width="1109" alt="Screenshot 2022-02-02 at 10 56 56" src="https://user-images.githubusercontent.com/71266765/152140999-bb6dbc1f-8595-44d4-aebf-dd74aa94a169.png">

<img width="1109" alt="Screenshot 2022-02-02 at 10 54 50" src="https://user-images.githubusercontent.com/71266765/152141004-ce8a8b44-4690-40df-b463-9eb022bbb7be.png">

Reply:
I think the existing issues you're seeing with IE8/9 might be limited to the component guide pages for some reason. Checking a [live example](https://www.gov.uk/learn-to-drive-a-car) shows IE8 with full technical support and minor visual issues and IE9 with full support.

**IE8:**

![Screenshot 2022-02-02 at 12 06 28](https://user-images.githubusercontent.com/861310/152150791-8a23e8ab-809c-425a-9988-c8e45885c493.png)

**IE9:**

![Screenshot 2022-02-02 at 12 06 52](https://user-images.githubusercontent.com/861310/152150836-66521e58-daee-41ee-82a9-91ba030e4b89.png)